tests/e2e/workloads/bdi/test_bdi_workload.py

```diff
@@ -0,0 +1,59 @@
+import ocs_ci.ocs.bdi.config as bdi_config
+from ocs_ci.ocs.bdi.bdi_base_class import TestBdiWorkloadBaseClass
```

Comment:
I see that running BDI requires 3 additional worker nodes and we are increasing the machineset in case of IPI. Should we skip it on vmware until we handle vmware node addition in the script.
Please also skip other relevant deployments also (BM, LSO)

Reply:
Added skipif for BM & LSO.
Also added ipi_deployment_required as it should run on aws & vsphere IPI